python/vyos/vpp/config_verify.py

```diff
@@ -102,6 +102,18 @@ def verify_vpp_remove_xconnect_interface(config: dict):
             )
 
 
+def verify_vpp_tunnel_source_address(config: dict, address: str):
```

Comment:
Would it not be easier to get `address` from config right here in the function? It is a part of `config` anyway.